### PR TITLE
feat: verifyKeyFastify

### PR DESCRIPTION
**Pull request by gcandal**

### CODE REVIEW

File: js/fastify/index.js [hunk 1]
```diff
@@ -1,2 +1,3 @@
 export * from "./plugins";
+export * from "./auth";
```
Comment (joamag) [inline js/fastify/index.js:2]: sort alphabeticly

File: js/fastify/plugins/acl.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ import { getRipeAuth } from "../../auth";
 const acl = async (app, options) => {
     app.addHook("onRequest", (req, res, next) => {
         req.getAcl = async ctx => {
+            req.ripeCtx = {};
```
Comment (joamag) [inline js/fastify/plugins/acl.js:8]: maybe `req.ripeCtx = req.ripeCtx === undefined ? {} : req.ripeCtx`

File: types/fastify/index.d.ts [hunk 3]
```diff
@@ -1,2 +1,3 @@
 export * from "./plugins";
+export * from "./auth";
```
Comment (joamag) [inline types/fastify/index.d.ts:2]: sort

**Review by joamag — CHANGES_REQUESTED** [review]

File: js/fastify/index.js (hunk 1 above)
Comment (gcandal) [reply js/fastify/index.js:2]: Plugins is a folder, that's why I placed it over there. Sort anyway?

File: types/fastify/index.d.ts (hunk 3 above)
Comment (gcandal) [reply types/fastify/index.d.ts:2]: Plugins is a folder, that's why I placed it over there. Sort anyway?

**Pull request merged**